### ISSUE TITLE
Chapter artwork not available with mobile data/roaming?

### ISSUE THREAD

**Issue by fredsleeve** [Podcast-Republic/PRApp#469]
Is it designed that chapter artwork won't load when using mobile data/roaming? All "Wifi only" options are disabled. Switching back to Wifi loads the chapter artwork.

**Comment by gemiren** [comment]
Chapter artwork loading uses the same WiFi control option as 'Streaming podcast via WiFi only'.

**Comment by fredsleeve** [comment]
I have definitely problems loading chapter artworks when switching the network from wifi to roaming with streaming. Switching back to wifi it works, but not always. Tested it with the German podcast "Stay Forever". Downloading the episode doesn't bring the chapter artwork back. Please take a look at it if it's reproducable for you.

**Comment by gemiren** [comment]
Will look into it and fix it asap. Thanks.

**Comment by gemiren** [comment]
I did some tests with the podcast "Stay Forever". I was able to get the chapter artwork displayed using either WiFi or mobile data connection. Please see the screen recording here https://www.dropbox.com/scl/fi/neiru41segf50mc3jst42/chapter-artwork-display.webm?rlkey=o8osvxez3era0px623a9uyfja&dl=0. Thanks.

**Comment by fredsleeve** [comment]
Please have a look at my screeen recording: https://c.gmx.net/@334288198261801446/6PYA-LtWTUeedmsI9G0OVQ
As soon as I switch to mobile network and go to the next episode chapter artworks disappears for the next loaded (Planetfall) and also previous loaded episode (Zelda). It also doesn't load, I throughout the whole episode when on mobile network. So it's not "loading issue" for me. Switching back to wifi brings it back. It does also appears when not using the play next/previous buttons.

**Comment by gemiren** [comment]
Can you double check whether you have the 'app Settings > Data & WiFi usage > Load artwork via WiFi only' unchecked? If that option is enabled then the chapter artwork won't be loaded when WiFi is not available.

**Comment by fredsleeve** [comment]
Yes, "Load artwork via WiFi only" is uncheck. I also checked and unchecked it again, to exclude an issue that the option is not correct set. Still doesn't work.

**Comment by fredsleeve** [comment]
Did some tests after #511 was fixed. Chapter images still not loading when using mobile data even when the episode have been downloaded. After switching back to wifi chapter images are loaded.

**Comment by fredsleeve** [comment]
Seems to be fixed
